lib/web_ui/test/engine/view_embedder/display_dpr_stream_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@TestOn('browser')
+library;
```

Comment:
I don't think these are needed, are they?

Reply:
I copied them from another test, I can remove them and see what happens. I don't want this to be tested in a VM at all.

Reply:
(Removed to see what happens!)

Reply:
All green, so it can be gone forever. Thanks for flagging this @mdebbar!